mods/creative/inventory.lua

```diff
@@ -128,23 +139,20 @@ function creative.register_tab(name, title, items)
 			if fields.creative_clear then
 				inv.start_i = 0
 				inv.filter = ""
-				creative.update_creative_inventory(player_name, items)
```

Comment:
It's actually a fix. `update_creative_inventory` is already called inside the `get` function provided by `sfinv`. So it was calling `update_creative_inventory` twice needlessly.